templates/system/header.html.twig

```diff
@@ -49,11 +49,6 @@
         <a href="https://my.croydon.gov.uk">My Account</a><i class="fas fa-user ml-2 mr-3" aria-hidden="true"></i>
       </div>
 
-      <div class="d-none d-lg-block">
-        <div class="form-inline">
-        {{ page.mobile_search }}
```

Comment:
It's good that you have dropped the mobile_search region from the header template.  If we don't need the mobile_search region anymore, how about removing it from the following files as well?:
- [localgov_theme_croydon.info.yml](https://github.com/localgovdrupal/localgov_theme_croydon/blob/410485f2bd3d633ea87cc42e017c488ed9185450/localgov_theme_croydon.info.yml#L25)
- [templates/system/page--localgov-campaigns-page.html.twig](https://github.com/localgovdrupal/localgov_theme_croydon/blob/410485f2bd3d633ea87cc42e017c488ed9185450/templates/system/page--localgov-campaigns-page.html.twig#L33)

Reply:
@Adnan-cds for this file - templates/system/page--localgov-campaigns-page.html.twig it imports the header and footer so should be showing the new single search